tools/make-keyboard-text/src/main/resources/values-hy/donottranslate-more-keys.xml

```diff
@@ -23,30 +23,35 @@
          U+0532: "Բ" ARMENIAN CAPITAL LETTER BEN
          U+0533: "Գ" ARMENIAN CAPITAL LETTER GIM -->
     <string name="keylabel_to_alpha">&#x0531;&#x0532;&#x0533;</string>
-    <!-- U+055E: "՞" ARMENIAN QUESTION MARK
```

Comment:
Corrected the punctuation for Armenian, left comments in Armenian and English where it's translatable to English.